app/code/community/ShipStream/Sync/Model/Cron.php

```diff
@@ -90,7 +101,34 @@ protected function _getTargetInventory(array $skus)
             ->where('si.stock_id = ?', Mage_CatalogInventory_Model_Stock::DEFAULT_STOCK_ID)
             ->where('p.sku IN (?)', $skus);
 
-        // TODO - subtract processing that is not "submitted"
+        return $db->fetchAssoc($select);
+    }
+
+    /**
+     * Retrieve Magento order items qty that are in processing state and not submitted to shipstream
+     * @param array $sku
+     * @return array
+     */
+    protected function _getProcessingOrderItemsQty(array $skus): array
```

Comment:
Please remove type declarations, even though 7.3 is EOL it may still be broadly in use.
```suggestion
    protected function _getProcessingOrderItemsQty(array $skus)
```